services/movie.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+
+from db.models import Genre, Actor, Movie
+
+
+def get_movies(
+        genres_ids: list[Genre] = None,
+        actors_ids: list[Actor] = None

```

Comment:
maybe i dont get it but imo this should not be hinted as "list of Genre instances", since inside the function you are clearly comparing them to ids but not instances....